### ISSUE TITLE
Support different namespace

### ISSUE THREAD

**Issue by timri** [robrichards/wse-php#31]
Hi, I had to consume a webservice that required WS-A, so I tried your library.
However, I was only successful after I changed the namespace:
```diff
diff -ruw wse-php_orig/src/WSASoap.php wse-php/src/WSASoap.php
--- wse-php_orig/src/WSASoap.php        2017-03-09 16:42:34.000000000 +0000
+++ wse-php/src/WSASoap.php     2017-07-19 10:46:54.672342000 +0000
@@ -48,7 +48,8 @@
  */
 class WSASoap
 {
-    const WSANS = 'http://schemas.xmlsoap.org/ws/2004/08/addressing';
+    const WSANS = 'http://www.w3.org/2005/08/addressing';
     const WSAPFX = 'wsa';
     private $soapNS, $soapPFX;
     private $soapDoc = null;
```
(To be of general use, this should probably be configurable, but the above change was enough to work for me)
Regards

**Comment by scoutkirkolson** [comment]
We're having the exact same problem. Since WSANS is a constant and it is used with self:WSANS (instead of static) we can't just override the class. Please look into this issue and make it configurable like **timri** said.

**Comment by rvdbogerd** [comment]
Duplicate of #46 and #33

**Comment by robrichards** [comment]
Fixed with PR#46
